ui-ngx/src/assets/locale/locale.constant-en_US.json

```diff
@@ -1116,7 +1118,11 @@
         "address2": "Address 2",
         "zip": "Zip",
         "phone": "Phone",
-        "label": "Label"
+        "label": "Label",
+        "transport": "Transport",
```

Comment:
Remove these fields. They are stored as fields in the database and also duplicated as telemetry. We should use telemetry in the dashboards because it has runtime updates.